src/components/Login/EnterPasswordPanel.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { useRouter } from 'next/router'
+
+import Form from '@/components/Login/LoginForm'
+import Button from '@/components/UI/button'
+
+const EnterPasswordPanel = (props: {
+  phoneNumber: string
+  togglePanel?: () => void
+}) => {
+  const [password, setpassword] = useState('')
+  const router = useRouter()
+
+  function handleSubmit(event: React.ChangeEvent<HTMLFormElement>) {
+    event.preventDefault()
+    event.target.reset()
+    setpassword('')
+  }
+
+  function handleForgot(
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
```

Comment:
This function doesn't need to take an event because the button that calls it is not a `submit` type.